### PR TITLE
submission markdown for Justin Pease idea box project

### PR DESCRIPTION
**Pull request by Jpease1020**

### CODE REVIEW

File: 1510/module_4_assigments/ideabox2.0/justin_pease.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Ideabox 2.0 Submission Form - Toni Rib
```
Comment (rrgayhart) [inline 1510/module_4_assigments/ideabox2.0/justin_pease.md]: You got Toni's name!

File: 1510/module_4_assigments/ideabox2.0/justin_pease.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+### Which extensions, if any, did you complete?
+
+I completed the inline editing.  A user can edit the title or body of an idea in real time without the page reloading.  The changes are send over to the database and the updates are displayed if you refresh the page.
+I also completed the sorting function.  A user can sort the ideas with the genius ideas defaulting to the top of the page.
+
+### Attach a .gif, or images of any extensions work being used on the site.
+
+Editing Inline:
+
+![](http://recordit.co/LJrMGA0gOh)
```
Comment (rrgayhart) [inline 1510/module_4_assigments/ideabox2.0/justin_pease.md]: To inline link recordit files, you have to grab the link with the `.gif` file extension at the end. What you have linked here is the video - but if you look on the bottom right, there's a gif button on it and if you click that, you should get the correct link!

File: 1510/module_4_assigments/ideabox2.0/justin_pease.md (hunk 1 above)
Comment (Jpease1020) [inline 1510/module_4_assigments/ideabox2.0/justin_pease.md]: Yea, I used her's as the template.

**Comment by rrgayhart** [comment]
Awesome, thanks!

**Pull request merged**